src/lib/support/Span.h

```diff
@@ -36,8 +37,16 @@ class Span
     constexpr explicit Span(const T (&databuf)[N]) : Span(databuf, N)
     {}
 
+    bool operator==(const Span<T> & other) const
```

Comment:
Here and throughout: of you just write `const Span & other` that will imply the same exact template args, no?

Reply:
updated. Thanks!